README.md

```diff
@@ -1 +1,70 @@
-# react-bookstore
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
```

Comment:
- [ ] Please, consider that this README file came with CRA (create-react-app), so you have to replace it with [MICROVERSE template](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍